api/metrics_test.go

```diff
@@ -64,71 +64,20 @@ func TestMetricsGet(t *testing.T) {
 
 	etreurl := server.url + etre.API_ROOT + "/entities/" + entityType + "?query=x"
 	statusCode, err := test.MakeHTTPRequest("GET", etreurl, nil, nil)
-	if err != nil {
-		t.Fatal(err)
-	}
-	if statusCode != http.StatusOK {
-		t.Errorf("response status = %d, expected %d", statusCode, http.StatusOK)
-	}
+	require.NoError(t, err)
+	assert.Equal(t, http.StatusOK, statusCode)
 
 	// GET /metrics
 	etreurl = server.url + etre.API_ROOT + "/metrics"
 	var gotMetrics etre.Metrics
 	statusCode, err = test.MakeHTTPRequest("GET", etreurl, nil, &gotMetrics)
-	if err != nil {
-		t.Fatal(err)
-	}
-
-	if statusCode != http.StatusOK {
-		t.Errorf("got HTTP status = %d, expected %d", statusCode, http.StatusOK)
```

Comment:
`assert.Equal(t, http.StatusOk, statusCode)` ?

Reply:
> `assert.Equal(t, http.StatusOk, statusCode)` ?

It's on line 76.